libraries/chain/include/eosio/chain/webassembly/eos-vm-oc/intrinsic_mapping.hpp

```diff
@@ -277,11 +277,7 @@ inline constexpr auto get_intrinsic_table() {
       "env.bls_pairing",
       "env.bls_g1_map",
       "env.bls_g2_map",
-      "env.bls_fp_mod",
-      "env.bls_verify",
```

Comment:
At the first look, I thought `bls_verify` was removed by accident as it did not have `aggregate` as a part of its name. But you were correct.